api/src/org/apache/cloudstack/api/command/admin/usage/GetUsageRecordsCmd.java

```diff
@@ -111,6 +111,30 @@ public Long getProjectId() {
     public String getUsageId() {
         return usageId;
     }
+    public void setAccountName(String accountName) {
+        this.accountName = accountName;
+    }
+
+    public void setDomainId(Long domainId) {
+        this.domainId = domainId;
+    }
+
+    public void setEndDate(Date endDate) {
+        this.endDate = endDate;
```

Comment:
Since ``Date`` instances are mutable, the ``this.endDate`` should be a copy of ``endDate`` to avoid side-effects.